tsconfig.json

```diff
@@ -1,16 +1,13 @@
 {
+  "extends": "@tsconfig/recommended/tsconfig.json",
   "compilerOptions": {
     "allowJs": true,
     "allowSyntheticDefaultImports": true,
     "declaration": true,
-    "declarationDir": "dist",
-    "esModuleInterop": true,
     "jsx": "react-jsx",
     "jsxImportSource": "@emotion/react",
     "lib": ["dom", "es2017"],
-    "noEmit": true,
-    "skipLibCheck": true,
-    "strict": true,
```

Comment:
Options removed here are in the base recommended config we're now extending, except for `noEmit`, which we no longer want as we're building our output with `tsc`.